pkg/morph/client/fee.go

```diff
@@ -33,19 +24,17 @@ func (x customFees) feeForMethod(method string) (fixedn.Fixed8, bool) {
 type fees struct {
 	defaultFee fixedn.Fixed8
 
-	customFees
-}
-
-// sets default fee for all operations.
-func (x *fees) setDefault(fee fixedn.Fixed8) {
-	x.defaultFee = fee
+	// customFees represents source of customized per-operation fees.
+	customFees map[string]fixedn.Fixed8
```

Comment:
Was the purpose of deprecating `customFees` type? If so, can we remove the type?